.github/workflows/ci.yml

```diff
@@ -10,49 +22,193 @@ env:
   CARGO_TERM_COLOR: always
   # Make sure CI fails on all warnings, including Clippy lints
   RUSTFLAGS: "-Dwarnings"
+  TBDEX_SDK_LOG_LEVEL: debug
```

Comment:
`TBDEX_SDK_LOG_LEVEL` should be called `WEB5_SDK_LOG_LEVEL` here

Reply:
nice catch, fixed!